zimports.py

```diff
@@ -649,6 +665,13 @@ def main(argv=None):
         action="store_true",
         help="If set, multiple imports can exist on one line"
     )
+    parser.add_argument(
+        "--max-line-length",
+        type=int,
+        default=config["flake8"].get("max-line-length", 200),
```

Comment:
how about 100?   I use 79 myself however > 100 is not recommended.  Also this should be the only place that any number is hardcoded.